lib/mac_layer/pdu/mpdu.cpp

```diff
@@ -13,6 +13,7 @@
 
 #include "mpdu.hpp"
 
+#define MPDU_DEBUG 0 // set to 1 for debug output
```

Comment:
Should this be in a header?

Reply:
I am never really sure, but in most cases my "DEBUG" defines are so I can see what's going on for alg dev and test. In terms of runtime debugging, my debug code is not that helpful. Runtime debug should be a LOG level IMHO, and that I have not put in, so I wouldn't want someone thinking otherwise when they include the header. 

Happy change if you tell me.

Reply:
We could think of some sort of os_log in osal.[h,c], Perhaps another PR, we can leave this for now.